docs/src/description/syntax.md

```diff
@@ -6,31 +6,54 @@ This page specifies the basic syntax, types, and keywords of AirScript.
 - `def`: used to [define the name](./structure.md) of an AirScript module.
-- `boundary_constraints`: used to declare the [source section](./structure.md#source-sections) where the [boundary constraints are described](./constraints.md#boundary_constraints).
-  - `first`: used to access the value of a trace column at the first row of the trace. _It may only be used when defining boundary constraints._
-  - `last`: used to access the value of a trace column at the last row of the trace. _It may only be used when defining boundary constraints._
-- `enf`: used to describe a single [constraint](./constraints.md).
-- `public_inputs`: used to declare the [source section](./structure.md#source-sections) where the [public inputs are declared](./declarations.md). _They may only be referenced when defining boundary constraints._
-- `periodic_columns`: used to declare the [source section](./structure.md#source-sections) where the [periodic columns are declared](./declarations.md). _They may only be referenced when defining integrity constraints._
+- `const`: used to declare [constants](./declarations.md#constant-constant).
 - `trace_columns`: used to declare the [source section](./structure.md#source-sections) where the [execution trace is described](./declarations.md).
   - `main`: used to declare the main execution trace.
   - `aux`: used to declare the auxiliary execution trace.
+- `public_inputs`: used to declare the [source section](./structure.md#source-sections) where the [public inputs are declared](./declarations.md). _They may only be referenced when defining boundary constraints._
+- `periodic_columns`: used to declare the [source section](./structure.md#source-sections) where the [periodic columns are declared](./declarations.md). _They may only be referenced when defining integrity constraints._
+- `random_values`: used to declare the [source section](./structure.md#source-sections) where the [random values are described](./declarations.md).
+- `boundary_constraints`: used to declare the [source section](./structure.md#source-sections) where the [boundary constraints are described](./constraints.md#boundary_constraints).
+  - `first`: used to access the value of a trace column at the first row of the trace. _It may only be used when defining boundary constraints._
+  - `last`: used to access the value of a trace column at the last row of the trace. _It may only be used when defining boundary constraints._
 - `integrity_constraints`: used to declare the [source section](./structure.md#source-sections) where the [integrity constraints are described](./constraints.md#integrity_constraints).
+- `enf`: used to describe a single [constraint](./constraints.md).
+- `let`: used to declare intermediate variables.
+- `\$`&lt;identifier&gt;: used to access random values provided by the verifier.
+- `\$main`: used to access columns in the main execution trace.
+- `\$aux`: used to access columns in the auxiliary execution trace.
```

Comment:
It's rendering correctly for me. Not sure what the issue could be. Could you please check once more since this section has been moved to another file so maybe that helps?